<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>jourview</class>
 <widget class="QWidget" name="jourview" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Lucida Console</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Journal</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="nr_txt" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" >
            <string>tilbuds nr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nr" >
           <property name="text" >
            <string>12000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="dato" >
         <property name="minimumSize" >
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text" >
          <string>00/00/0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" >
            <string>SÃ¦lger ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sid" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" >
            <string>rkj</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="Luk_lab" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>70</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" >
            <string>Lukke Pro</string>
           </property>
           <property name="textFormat" >
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents" >
            <bool>false</bool>
           </property>
           <property name="indent" >
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="luk" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>87</width>
             <height>22</height>
            </size>
           </property>
           <property name="autoFillBackground" >
            <bool>false</bool>
           </property>
           <property name="value" >
            <number>10</number>
           </property>
           <property name="textVisible" >
            <bool>true</bool>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance" >
            <bool>false</bool>
           </property>
           <property name="textDirection" >
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="Title" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Lucida Console</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="nota_text" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="mouseTracking" >
            <bool>true</bool>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="contextMenuPolicy" >
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="acceptDrops" >
            <bool>false</bool>
           </property>
           <property name="autoFillBackground" >
            <bool>true</bool>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy" >
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="tabChangesFocus" >
            <bool>false</bool>
           </property>
           <property name="undoRedoEnabled" >
            <bool>false</bool>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>Luk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="helhed.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>jourview</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>204</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel" >
     <x>203</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
